internal/app/mistapiconnector/stats_collector.go

```diff
@@ -25,8 +25,8 @@ type metricsCollector struct {
 }
 
 func startMetricsCollector(ctx context.Context, period time.Duration, nodeID, ownRegion string, mapi *mist.API, producer *event.AMQPProducer, amqpExchange string, infop infoProvider) {
-	census.IncMultistreamBytes(0)
```

Comment:
Not sure why/if this initialization of 0 bytes is needed. Since there is no streamID at this point we are pushing the zero value string `""`

Reply:
I think in this case initialisation should be removed because it creates weird datapoint with empty streamID.

Reply:
That sounds good, any reason it required the initialization before?

Reply:
Because without initialisation Grafana doesn't show anything for the metric, so it's hard to tell if metric just not been measured yet or there is some problem in scraping.